test/graphtilebuilder.cc

```diff
@@ -110,7 +110,10 @@ TEST(GraphTileBuilder, TestDuplicateEdgeInfo) {
   // add edge info for node 0 to node 1
   bool added = false;
   test.AddEdgeInfo(0, GraphId(0, 2, 0), GraphId(0, 2, 1), 1234, 555, 0, 120,
-                   std::list<PointLL>{{0, 0}, {1, 1}}, {"einzelweg"}, {"1xyz tunnel"}, 0, added);
+                   std::list<PointLL>{{0, 0}, {1, 1}}, {"einzelweg"},
+                   {"1xyz tunnel",
+                    std::string(1, static_cast<char>(TaggedValue::kZLevel)) + char(-1)},
```

Comment:
honestly i would shy away from updating these t ypes of tests and instead focus on gurka. the gurka tests do the entire process, not just pieces of hte process like these and they are less error prone. if it seems like gurka wont let you express the thing you want to express lets give gurka that functionality and let these old school tests alone if we can

Reply:
So probably tests which we have in `test_tagged_values` is enough?

Reply:
yes, i would feel free to  revert this. i hope that one day gurka will have so much in it we can delete these tests completely

Reply:
actually that would be a nice experiment. disable the mjolnir old style tests and see how much coverage drops (or even better where it drops)